Package.swift

```diff
@@ -1322,13 +1322,13 @@ func abseilDependency() -> Package.Dependency {
   // as the headers in the binary version of abseil are unusable.
   if ProcessInfo.processInfo.environment["FIREBASE_SOURCE_FIRESTORE"] != nil {
     packageInfo = (
-      "https://github.com/firebase/abseil-cpp-SwiftPM.git",
-      "0.20220623.0" ..< "0.20220624.0"
+      "https://github.com/abseil/abseil-cpp.git",
+      "20211102.0" ..< "20211103.0"
```

Comment:
@paulb777 To quickly test this I switched to the https://github.com/abseil/abseil-cpp.git repo instead of https://github.com/firebase/abseil-cpp-SwiftPM.git (which doesn't have `20211102`). Merging this would degrade clone times somewhat when using SPM. I'll see if I can find some instructions for how to proceed.

Reply:
Oh, that explains why I didn't see the problem today that I saw yesterday. 😛 

Yep, ideally we should get the abseil binary. the instructions are around somewhere ....

Reply:
Just to clarify, the abseil binary repo was fine (https://github.com/google/abseil-cpp-binary/tree/1.2021110200.0). The https://github.com/firebase/abseil-cpp-SwiftPM.git repo is used for source builds of Firestore.

Reply:
We discussed offline and for this release we will use the https://github.com/abseil/abseil-cpp.git repo since it will only affect clone times when using the Firestore source distribution (hidden behind the `FIREBASE_SOURCE_FIRESTORE` environment variable).

Note: When we previously used the 1.44.0 gRPC release in Firebase 10.2, we had a mismatch in the abseil version between SPM and CocoaPods. Context: https://github.com/firebase/firebase-ios-sdk/blob/a1b78b247b11e5be77836c6ac86b66b453cb101d/Package.swift#L178
This is why we don't have `20211103` tagged in the https://github.com/firebase/abseil-cpp-SwiftPM.git repo.